arch/renesas/src/rx65n/rx65n_irq.c

```diff
@@ -426,6 +426,77 @@ void up_disable_irq(int irq)
 
 #endif
 #endif
+#ifdef CONFIG_RX65N_RIIC0
+  if (irq == RX65N_RIIC0_RXI0_IRQ)
+    {
+      ICU.IER[6].BIT.IEN4 = 0;
```

Comment:
Firstly, it is observed that output binary image size is smaller when bit fields are used in RX65N compilation. Secondly, bit fields are generally used in Renesas drivers as part of their FIT driver package releases. We feel that keeping the same coding style would help users in porting across platforms. Because of these advantages we would like to retain the coding style if it is not violating any NuttX guidelines.    

Discussion on the same topic was done on NuttX google group some time back and the discussion link is given below:

https://groups.google.com/forum/#!searchin/nuttx/rx65n%7Csort:date/nuttx/jwV1n6oiDQQ/uMUFtF7OBgAJ

Reply:
Ok, I recall this discussion.